code/modules/modular_computers/file_system/programs/civilian/cargo_control.dm

```diff
@@ -7,6 +7,7 @@
 	available_on_ntnet = 1
 	required_access_download = access_hop
 	required_access_run = access_cargo
+	usage_flags = PROGRAM_CONSOLE | PROGRAM_LAPTOP
```

Comment:
The Cargo Control application should only be available from stationary consoles for now.
With some upcoming changes it will be made available on tablets aswell (as a read only system)

Reply:
Before this they were available everywhere. But make them avavible on telescreens as they could be used as alternatives to consoles.